Python/RSA_Key_Generation/README.md

```diff
@@ -2,15 +2,33 @@
 ## This script is used to create a private/public keypair for SSH-authentication for password-less connecting to a remote server.
 
 + On running this script, will create two files with (.pem) format which contains public and private key.
-+ Defined length of keys are 2048 bits *(recommended size)* which determines the strength of keys.
++ Defined length of keys are 4096 bits *(recommended size)* which determines the strength of keys.
 
 ## To run the script:
 1. Install the dependencies by running following command in terminal.
 
    `pip install pycryptodome`
+
 2. Run the script.
 
-    `python rsa_key_generation.py`
+    `python RSA.py`
+
+    The script runs until you enter valid commands- 
+    1) generate_keys \<folder path to store keys\>
+    
+     \(eg - `generate_keys ./keys`\)
+
+    2) encrypt \<path to public key file\> \<file to encrypt\> \<encrypted file name\> 
+    
+     \(eg - `encrypt ./keys/public_key.pem nature.jpeg enc_script`\)
```

Comment:
Remove the image file from the repo. Ask the user to add an image file here and make it dynamic so that we don't hardcode it

Reply:
The code is dynamic only. The user needs to enter the path of the file which he wants to encrypt in command. The .jpeg and .txt files are only for example.